tee-worker/cli/src/trusted_base_cli/commands/litentry/set_scheduled_mrenclave.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020-2023 Litentry Technologies GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{
+	command_utils::mrenclave_from_base58,
+	get_layer_two_nonce,
+	trusted_cli::TrustedCli,
+	trusted_command_utils::{get_identifiers, get_pair_from_str},
+	trusted_operation::perform_trusted_operation,
+	Cli,
+};
+use codec::Decode;
+use ita_stf::{Index, TrustedCall, TrustedGetter, TrustedOperation};
+use itp_stf_primitives::types::KeyPair;
+use itp_types::SidechainBlockNumber;
+use log::*;
+use sp_core::Pair;
+use std::boxed::Box;
+
+#[derive(Parser)]
+pub struct SetScheduledMrenclaveCommand {
+	bn: SidechainBlockNumber,
+	mrenclave: String,
+}
+
+impl SetScheduledMrenclaveCommand {
+	pub(crate) fn run(&self, cli: &Cli, trusted_cli: &TrustedCli) {
+		let root = get_pair_from_str(trusted_cli, "//Alice");
+
+		let (mrenclave, shard) = get_identifiers(trusted_cli);
+		let nonce = get_layer_two_nonce!(root, cli, trusted_cli);
+
+		let top: TrustedOperation = TrustedCall::set_scheduled_mrenclave(
+			root.public().into(),
+			self.bn,
+			mrenclave_from_base58(&self.mrenclave),
```

Comment:
Maybe we should expect a hex string here - just to be consistent with the extrinsic in teerex

Reply:
done